src/main/java/com/google/api/generator/gapic/composer/Composer.java

```diff
@@ -29,16 +30,12 @@
 import com.google.api.generator.gapic.composer.resourcename.ResourceNameHelperClassComposer;
 import com.google.api.generator.gapic.composer.rest.HttpJsonServiceCallableFactoryClassComposer;
 import com.google.api.generator.gapic.composer.rest.HttpJsonServiceStubClassComposer;
-import com.google.api.generator.gapic.model.GapicClass;
-import com.google.api.generator.gapic.model.GapicContext;
-import com.google.api.generator.gapic.model.GapicPackageInfo;
-import com.google.api.generator.gapic.model.Sample;
-import com.google.api.generator.gapic.model.Service;
-import com.google.api.generator.gapic.model.Transport;
+import com.google.api.generator.gapic.model.*;
```

Comment:
Seems like the IDE defaulted to use a star for a bunch of changed files

Reply:
Thanks for catching! I reverted.